packages/ui/src/components/va-affix/VaAffix.vue

```diff
@@ -18,6 +18,7 @@
 <script lang="ts">
```

Comment:
`VaButtonToggle` `color` and `border-color` falls back to white color after selection.

![ButtonToggleWhiteDefault](https://user-images.githubusercontent.com/7745899/163006103-558964bf-d6f5-4224-a70c-d6368402ff84.jpg)

Reply:
Fixed in [eacc354](https://github.com/epicmaxco/vuestic-ui/pull/1634/commits/eacc354057588314777b4ed8fb17fdbebe52796b)

Reply:
I guess link was removed after force push.